src/fundus/publishers/de/tagesspiegel.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+from typing import Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class TagesspiegelParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div[id=story-elements] > p")
+        _subheadline_selector = CSSSelector(".Ha6")
```

Comment:
It looks like the newspaper also supports a `summary`.

Further the subheadline selector does not seem to work for most cases.

Reply:
added summary and fixed the subheadline selector